### ISSUE TITLE
Allow basic calculations when entering expressions like `=12*78`

### ISSUE THREAD

**Issue by AlexandreBonneau** [autoNumeric/autoNumeric#542]
It would be useful to allow users to enter simple calculation expression of the format `=expression`, and on `blur` or a defined key sequence (`Enter`?), the result of this expression would be `set()` and formatted.

Examples:
- `=12+34` -> `46,00 €`
- `=-5.7*60` -> `-342,00 €`
- `=6000/45*101.45` -> `13.526,66 €`
- `=(78-34)*18.78+24000` -> `24.826,32 €`
- `=28.45*0.15/100` -> `0,042675`

What are you thoughts on the difficulty to implement this?
Do you think creating a parser would add many lines of code to the codebase?

**Comment by alexturpin** [comment]
https://github.com/josdejong/mathjs supports doing this. I know AutoNumeric has no dependencies as of yet but it seems like a safer bet to me to use a well established library rather than reinventing the wheel for this.

**Comment by AlexandreBonneau** [comment]
I've seen `mathjs`, however I'm not keen on adding a dependency to AutoNumeric, specially if that feature could be deactivated with an option.
It would then not really be convenient to depend on a third party library the user would not use.
Also, `mathjs` is very, very advanced and we do not need all that complexity.
We just need additions, multiplications and divisions, and perhaps parenthesis and percentages too.

I've started to work on that feature few months ago, but did not had the time to finish it yet (because even we used the mathjs parser and evaluator, we still need to manage all the interactions with the 'normal' AutoNumeric behavior (undo/redos, blur/mouseover events, cut/paste, drop event, alt key, etc.)).
Any help is welcome ;P :sunglasses:
